### PR TITLE
Welcoming contact field

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,127 +0,0 @@
-# Contributor Covenant Code of Conduct
```
Comment (VishnuSanal) [inline CODE_OF_CONDUCT.md:1]: Why this got deleted? Do any of Weblate's policies requires us to do so? Or is it just for brevity?

**Review by VishnuSanal — COMMENTED** [review]
@comradekingu Thanks for pointing this out and taking the effort to make this PR. Please see the comments.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (comradekingu) [reply CODE_OF_CONDUCT.md:1]: I deleted it so I can contribute in good conscience.
Weblate is libre software. It neither has a CoC, nor enforces one as such.
In the interest of brevity, I think it does nothing productive vis-à-vis having a clear contact field option.

Comment (VishnuSanal) [reply CODE_OF_CONDUCT.md:1]: > I deleted it so I can contribute in good conscience. Weblate is libre software. It neither has a CoC, nor enforces one as such. In all manners pertaining to brevity, I think a this CoC does nothing productive vis-à-vis a clear contact field.

Okay, makes sense. We defenitely can have CONTACT.md. But, I don't think there's a need to remove CODE_OF_CONDUCT.md for this. Can you please revert the deletion? Thanks

File: CONTACT.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Welcome to the Power Quotes community.
```
Comment (VishnuSanal) [inline CONTACT.md:1]: I think this can be "Welcome to the Quotes Status Creator community."

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (comradekingu) [reply CODE_OF_CONDUCT.md:1]: I don't want any contributions of mine in a project that has this CoC.

File: CONTACT.md (hunk 2 above)
Comment (comradekingu) [inline CONTACT.md:1]:
```suggestion
Welcome to the Quotes Status Creator community.
```

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (VishnuSanal) [reply CODE_OF_CONDUCT.md:1]: I think you are intending to say - we need only a pledge and a contact field and not any restrictions like bans as such on this OSS project. Right? Merging...

**Pull request merged**